wix-ecom-backend/wix-ecom-backend/guides/1-wix-ecom-backend-intro.md

```diff
@@ -3,27 +3,69 @@ title: Introduction
+The `wix-ecom-backend` Velo APIs are a comprehensive suite of services that address standard and specialized needs in online selling.
 
-## Wix eCommerce API and Wix Stores API
+The APIs serve as a foundational layer for Wix's own business solutions, such as [Stores](https://support.wix.com/en/article/wix-stores-about-wix-stores), [Bookings](https://support.wix.com/en/article/wix-bookings-about-wix-bookings), and [Restaurants Orders](https://www.wix.com/app-market/restaurants-orders-app).
 
-The main difference between the two APIs is that while **Wix eCommerce manages the cart, checkout, and order** phases of the online store experience, **Wix Stores continues to manage product catalogs, inventories and collections**. The following table outlines the differences between the two APIs.
+The `wix-ecom-backend` Velo APIs provide developers with a comprehensive set of services for customizing eCommerce functionality on Wix sites. The APIs empower developers to extend functionality, create features, and build robust eCommerce solutions that meet the specific needs of their users.
 
-| Element                                  | Previous Location                                | Current Location            |
-| -----------------------------------------|--------------------------------------------------|-----------------------------|
-| Cart                                     | [Wix Stores](https://www.wix.com/velo/reference/wix-stores)                                  | [Wix eCommerce Cart](https://www.wix.com/velo/reference/wix-ecom-backend/cart)
-| Checkout                                 | No Stores functionality                          | [Wix eCommerce Checkout](https://www.wix.com/velo/reference/wix-ecom-backend/checkout)
-| Order                                    | [Wix Stores Backend](https://www.wix.com/velo/reference/wix-stores-backend)                                       | [Wix eCommerce Orders](https://www.wix.com/velo/reference/wix-ecom-backend/orders)
-| Catalog                                  | [Wix Stores Backend](https://www.wix.com/velo/reference/wix-stores-backend)                                       | Wix Stores - no change
-| Inventory                                | [Wix Stores Backend](https://www.wix.com/velo/reference/wix-stores-backend)                                       | Wix Stores - no change
-| Collections                              | [Wix Stores Backend](https://www.wix.com/velo/reference/wix-stores-backend)                                       | Wix Stores - no change
+With the `wix-ecom-backend` Velo APIs, you can do the following and more:
 
-> **Note:** Further functionality will be coming soon.
++ Access, manage, and create a site visitor's [cart](https://www.wix.com/velo/reference/wix-ecom-backend/cart/introduction). The same functionality also applies, without the use of a cart ID, to a site visitor's [current cart](https://www.wix.com/velo/reference/wix-ecom-backend/currentcart/introduction).
```

Comment:
I get why you added current cart here, but it seems a little unwieldy. Maybe there's a way to shorten into one sentence? Something like "Access, manage and create any customer's [cart](), including a site visitor's [current cart](). (Especially since you elaborate on this below as well.)